blocks/header/header.js

```diff
@@ -606,7 +613,10 @@ const buildMenuContent = (menuData, navEl) => {
 
         list.classList.add(`${blockName}__category-items`);
         [...list.querySelectorAll('li')].forEach(rebuildCategoryItem);
-        [...list.querySelectorAll('a')].forEach((el) => el.classList.add(`${blockName}__link`));
+        [...list.querySelectorAll('a')].forEach((el) => {
+          const isInlineLink = el.classList.contains('inline-link');
+          el.classList[isInlineLink ? 'remove' : 'add'](`${blockName}__link`);
```

Comment:
No need to change, but this can also be done with toggle like this:
`el.classList.toggle(`${blockName}__link`, isInlineLink);`